PatientRegistrationApp/src/components/SinglePatientFollowUp.vue

```diff
@@ -1,7 +1,9 @@
 <template>
 	<v-container fluid>
+
+
     <v-row justify="center">
-  <v-col class="d-flex" cols="6" sm="6">
+    <v-col class="d-flex" cols="6" sm="6">
```

Comment:
To make the format tablet and mobile friendly, this seems to be a helpful adjustment:
<v-col class="d-flex" cols="10" sm="8" md="6" lg="6">